test/functional/sqlsrv/MsHelper.inc

```diff
@@ -449,7 +773,7 @@ function getSqlType($k)
     return ("udt");
 }
 
-function getDriverType($k, $dataSize)
+function getDriverType($k, $dataSize = 512)
```

Comment:
where does 512 come from?

Reply:
It was for convenience, but I only need it for one test, so I'll move this constant to that test instead.